nucypher/__init__.py

```diff
@@ -37,23 +39,24 @@
 
 from nucypher.config.constants import REPORT_TO_SENTRY, NUCYPHER_SENTRY_ENDPOINT, PACKAGE_NAME
 
-try:
-    import sentry_sdk
-    from sentry_sdk.integrations.logging import LoggingIntegration
-except ImportError:
-    if REPORT_TO_SENTRY is True:
-        raise ImportError("Sentry")
-else:
-    import logging
-    sentry_logging = LoggingIntegration(
-        level=logging.INFO,        # Capture info and above as breadcrumbs
-        event_level=logging.DEBUG  # Send debug logs as events
-    )
-    sentry_sdk.init(
-        dsn=NUCYPHER_SENTRY_ENDPOINT,
-        integrations=[sentry_logging],
-        release='{}@{}'.format(PACKAGE_NAME, __version__)
-    )
+if not hasattr(sys, '_pytest_is_running'):
+    try:
+        import sentry_sdk
+        from sentry_sdk.integrations.logging import LoggingIntegration
+    except ImportError:
+        if REPORT_TO_SENTRY is True:
+            raise ImportError("Sentry")
```

Comment:
I think, it's enough to just `raise` here, to reraise the original error (which is, imo, slightly more informative than just "Sentry")

Reply:
Moreover, do we even want to try to import `sentry_sdk` if `REPORT_TO_SENTRY` is `False`?

Reply:
Haha - This looks like an over-zealous cut!

Reply:
I think simply "Sentry" is an incredible and gripping error message.  It sounds like a Stephen King title.  I know I'll think twice about doing anything that might risk raising this.

Reply:
> Moreover, do we even want to try to import sentry_sdk if REPORT_TO_SENTRY is False?

For testnet - I'm leaning into it!  Only slight differences in flow control in either case.

Reply:
> Haha - This looks like an over-zealous cut!

I can't see evidence of this. I think the PR is failing CI due to the `pytest_configure` and `pytest_unconfigure` methods.

Reply:
@tux sounds like you are talking about two sperate Issues.  The Error message above is too terse.